frontend/public/components/cluster-settings/oauth.tsx

```diff
@@ -58,6 +58,7 @@ const OAuthDetails: React.SFC<OAuthDetailsProps> = ({obj}: {obj: OAuthKind}) =>
     htpasswd: 'HTPasswd',
     ldap: 'LDAP',
     oidconnect: 'OpenID Connect',
+    basicauth: 'Basic Authentication',
```

Comment:
I'd move this to the top to keep the list alphabetized

Reply:
@spadgett good point it was alpha when it was Remote :)